### PR TITLE
Docs: Add badges to docusaurus home.md

### LINKED ISSUE #224: Docs: Add badges to docusaurus home.md file
### Is your feature request related to a problem? Please describe.

Add badges to docusaurus home.md file.

### Describe the solution you'd like

image

### Additional context

[shields.io](https://shields.io/) can be used for generating the badges.

### PR DESCRIPTION
**Pull request by naveen8801**
### Description

After changes , Home.md will look like this .

Screenshot (48)

Fixes #224 

### Type of Change:

- Documentation

**Code/Quality Assurance Only**

- This change requires a documentation update

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by decon-harsh** [comment]
LGTM @naveen8801

File: docs/docs/Home.md [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/anitab-org/bridge-in-tech-backend/pulse)
+[![PR's Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat)](https://github.com/anitab-org/bridge-in-tech-backend/blob/develop/.github/PULL_REQUEST_TEMPLATE.md)  
+
 ---
+
 id: Home
 title: Home
 slug: /
+
 ---
```
Comment (Rahulm2310) [inline docs/docs/Home.md:6]: Can't add anything above the frontmatter. Please add the badges here i.e. below the frontmatter.

File: docs/docs/Home.md [hunk 2]
```diff
@@ -1,7 +1,14 @@
+[![GPLv3 License](https://img.shields.io/badge/License-GPL%20v3-yellow.svg)](https://github.com/anitab-org/bridge-in-tech-backend/blob/develop/LICENSE)
+[![Open Source](https://badges.frapsoft.com/os/v1/open-source.svg?v=103)](https://github.com/anitab-org/bridge-in-tech-backend)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/anitab-org/bridge-in-tech-backend/pulse)
+[![PR's Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat)](https://github.com/anitab-org/bridge-in-tech-backend/blob/develop/.github/PULL_REQUEST_TEMPLATE.md)  
+
 ---
+
```
Comment (Rahulm2310) [inline docs/docs/Home.md]: No need for these new lines. Please remove them as well.

**Comment by naveen8801** [comment]
hi @Rahulm2310  , on adding badges just below the frontmatter , the whole badges are converted to simple text ,on removing both the horizontal lines it become fine , so should i remove them ? ,

**Comment by Rahulm2310** [comment]
@naveen8801  Noo. Those horizontal lines are a part of the frontmatter. Can't remove those as it is required by docusaurus when rendering pages.
`````
----
id : home
slug : /
----
`````
`<Add the badges here>`

**Comment by naveen8801** [comment]
> @naveen8801 Noo. Those horizontal lines are a part of the frontmatter. Can't remove those as it is required by docusaurus when rendering pages.
> 
> ```
> ----
> id : home
> slug : /
> ----
> ```
> 
> `<Add the badges here>`

ok @Rahulm2310 👍

File: docs/docs/Home.md [hunk 3]
```diff
@@ -2,11 +2,12 @@
 id: Home
 title: Home
 slug: /
+---
 [![GPLv3 License](https://img.shields.io/badge/License-GPL%20v3-yellow.svg)](https://github.com/anitab-org/bridge-in-tech-backend/blob/develop/LICENSE)
 [![Open Source](https://badges.frapsoft.com/os/v1/open-source.svg?v=103)](https://github.com/anitab-org/bridge-in-tech-backend)
 [![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/anitab-org/bridge-in-tech-backend/pulse)
 [![PR's Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat)](https://github.com/anitab-org/bridge-in-tech-backend/blob/develop/.github/PULL_REQUEST_TEMPLATE.md)  
----
+
```
Comment (devkapilbansal) [inline docs/docs/Home.md:14]: Line should be blank. Remove white spaces

Comment (naveen8801) [reply docs/docs/Home.md:14]: @devkapilbansal , I'll do that 👍

**Comment by Rahulm2310** [comment]
@naveen8801 Can you add labels for `Zulip`, `Forks` and `Stars` as well. Also, please update the snapshot in the PR description.

**Comment by naveen8801** [comment]
> @naveen8801 Can you add labels for `Zulip`, `Forks` and `Stars` as well. Also, please update the snapshot in the PR description.

Hi @Rahulm2310  , I have implemented the changes , but why one test is falling ??

**Review by Rahulm2310 — COMMENTED** [review]
@naveen8801 Why are these changes made to config.py? Please remove it from the commit

File: config.py [hunk 4]
```diff
@@ -86,9 +86,9 @@ class LocalConfig(BaseConfig):
     DEBUG = True
 
     # Using a local postgre database
-    SQLALCHEMY_DATABASE_URI = "postgresql:///bit_schema"
+    # SQLALCHEMY_DATABASE_URI = "postgresql:///bit_schema"
 
-    # SQLALCHEMY_DATABASE_URI = BaseConfig.build_db_uri()
+    SQLALCHEMY_DATABASE_URI = BaseConfig.build_db_uri()
```
Comment (devkapilbansal) [inline config.py]: This is not desirable :thinking:

File: docs/docs/Home.md [hunk 5]
```diff
@@ -28,7 +36,6 @@ A primary goal of AnitaB.org is to be inclusive of the largest number of contrib
 
 See the full [Code of Conduct](https://github.com/anitab-org/bridge-in-tech-backend/blob/develop/.github/CODE_OF_CONDUCT.md) and the [Reporting Guidelines](https://github.com/anitab-org/bridge-in-tech-backend/blob/develop/.github/REPORTING_GUIDELINES.md).
 
-
 # Contact
 
-You can reach us at [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/#narrow/stream/237630-bridge-in-tech). This project is discussed on the [#bridge-in-tech](https://anitab-org.zulipchat.com/#narrow/stream/237630-bridge-in-tech) stream.
\ No newline at end of file
+You can reach us at [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/#narrow/stream/237630-bridge-in-tech). This project is discussed on the [#bridge-in-tech](https://anitab-org.zulipchat.com/#narrow/stream/237630-bridge-in-tech) stream.
```
Comment (devkapilbansal) [inline docs/docs/Home.md]: Why this is marked as changed? Please revert any white-space/tab changes here if done

File: docs/docs/Home.md (hunk 3 above)
Comment (devkapilbansal) [reply docs/docs/Home.md:14]: White space is yet to be removed

**Review by devkapilbansal — CHANGES_REQUESTED** [review]
@naveen8801 please look in these requests below

File: docs/docs/Home.md (hunk 3 above)
Comment (naveen8801) [reply docs/docs/Home.md:14]: > White space is yet to be removed

Hi @devkapilbansal  , i am not able to trace any white space in my text editor :( can you tell me exactly in which line it is present ?

Comment (devkapilbansal) [reply docs/docs/Home.md:14]: It is not there now :thinking:

**Review by Rahulm2310 — APPROVED** [review]
LGTM :100:

**Review by HeroicHitesh — APPROVED** [review]
LGTM

**Review by devkapilbansal — APPROVED** [review]
LGTM :+1:

**Review by codesankalp — APPROVED** [review]

**Review by Amulya-coder — APPROVED** [review]

**Comment by mtreacy002** [comment]
@naveen8801 , just a friendly reminder. Please do not use `#`(hashtag)  inside your new pr branch name because the reviewer cannot easily pull the pr to test it locally. Since this is a documentation issue, it's not critical to have the PR tested. But in coding related changes when we need to test the PR, you would be requested to rename your branch and remove the `#` hashtag 😉. I'll approve this PR for now.😉.

**Review by mtreacy002 — APPROVED** [review]
Well done, @naveen8801. Thank you for accommodating to the change requested and contributing to the project 🎉🎉🎉.

**Comment by naveen8801** [comment]
> @naveen8801 , just a friendly reminder. Please do not use `#`(hashtag)  inside your new pr branch name because the reviewer cannot easily pull the pr to test it locally. Since this is a documentation issue, it's not critical to have the PR tested. But in coding related changes when we need to test the PR, you would be requested to rename your branch and remove the `#` hashtag 😉. I'll approve this PR for now.😉.

Thanks @mtreacy002 , I'll take care of it from next time . 😃

**Pull request merged**